tests/test_mesh.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+import trimesh
+import numpy as np
+import os
+
+import genesis as gs
+import genesis.utils.mesh as mu
+import genesis.utils.gltf as gltf_utils
+
+
+@pytest.mark.parametrize("glb_file", ["assets/cube.glb", "assets/combined_srt.glb", "assets/combined_transform.glb"])
+def test_glb_parse(glb_file):
+    print(f"Testing glb parsing...")
+    glb_file = os.path.join(os.path.dirname(__file__), glb_file)
```

Comment:
You can use `from genesis.utils.mesh import get_assets_dir` after changing the location of the assets